client/my-sites/post-type-list/post-type-list-post-actions/index.jsx

```diff
@@ -0,0 +1,32 @@
+/**
+ * External dependencies
+ */
+import React, { PropTypes } from 'react';
+
+/**
+ * Internal dependencies
+ */
+import EllipsisMenu from 'components/ellipsis-menu';
+import PostTypeListPostActionsEdit from './edit';
+import PostTypeListPostActionsPublish from './publish';
+import PostTypeListPostActionsTrash from './trash';
+import PostTypeListPostActionsView from './view';
+import PostTypeListPostActionsRestore from './restore';
+
+export default function PostTypeListPostActions( { globalId } ) {
+	return (
+		<div className="post-type-list-post-actions">
+			<EllipsisMenu position="bottom left">
+				<PostTypeListPostActionsView globalId={ globalId } />
```

Comment:
Could be a single boolean prop `includeDefaultActions={ false }`, and the developer could reincorporate any they'd want to keep in children?

```js
<PostTypeListPostActions includeDefaultActions={ false }>
	<PostTypeListPostActionsEdit />
	<MyCustomAction />
</PostTypeListPostActions>
```